readme.md

```diff
@@ -38,7 +45,7 @@ Describe the differences between a SQL and NoSQL DB, and when you might use each
 
 Your Answer:
 ```text
-
+Difference between SQL and noSQL is relational data base. in SQL, each dataset has relation to another. However, in noSQL, the data are all in one big dataset with key value pair format.
```

Comment:
Right, SQL databases (like postgreSQL) are relational and consist of tables of records defined by schema. NoSQL databases (like MongoDB) consist of documents in collections why by default are schema-less (we used mongoose for schemas).